iib/web/static/api_v1.yaml

```diff
@@ -1346,6 +1347,14 @@ components:
                 image pull specs configured in IIB_GRAPH_MODE_INDEX_ALLOW_LIST in the IIB API Config. If not
                 specified, "--mode" will not be added to OPM commands to add the bundle(s) to the index.
               default: None
+            ignore_bundle_ocp_version:
+              type: string
```

Comment:
shouldn't this be boolean? Also, I think it is only added in the response schema. As a user, if I am gonna submit a merge index request, I would check the properties/request schema rather than the response to see what parameters I can use when submitting a request. Let me know if I am missing something here!

Reply:
Yes, this should be boolean and I missed the request part. Thank you for catching it - fixed.